ParkingZoneApp/Areas/Admin/Controllers/ParkingSlotController.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using ParkingZoneApp.Services;
+using ParkingZoneApp.Services.ParkingSlotService;
+using ParkingZoneApp.ViewModels.ParkingSlotViewModels;
+
+namespace ParkingZoneApp.Areas.Admin.Controllers;
+
+[Authorize]
+[Area("Admin")]
+public class ParkingSlotController : Controller
+{
+    private readonly IParkingSlotService _slotService;
+    private readonly IParkingZoneService _zoneService;
+
+    public ParkingSlotController(IParkingSlotService _slot, IParkingZoneService _zone)
+    {
+        _slotService = _slot;
+        _zoneService = _zone;
+    }
+
+    #region Index
+    // GET: Admin/ParkingSlot
+    public IActionResult Index(int id)
+    {
+        var parkingSlotVMs = _slotService.GetSlots(id)
+                                         .Select(slot => new ListOfParkingSlotsVM(slot));
```

Comment:
So big space for Select

Reply:
If i put all that in one line then it will be hard to read